pkg/controller/workspace/k8s/convertToLowLevelObjects.go

```diff
@@ -1,4 +1,4 @@
-package workspace
+package k8s
```

Comment:
I understand the idea of moving this one package down. But I don't really understand the choice of the `k8s` package name. Seems a bit confusing to me.

Reply:
Agree, it's just the first word that came to my mind, the package convert Che specific objects to K8s native. I will think more and try to find more suitable word here.

Reply:
do you think `component` would be good enough name for this package?

Reply:
I like `component`, or maybe `infrastructure`/`cluster`?

Reply:
I like `component` the best I think. Or even `components` (with a `s`) ?

Reply:
Seems it's more common to use singular word in go package name, but I do not have strong opinion